source/documentation/other-topics/rotate-tokens-keys.html.md.erb

```diff
@@ -173,8 +179,29 @@ This guide covers only secrets provided by the Cloud Platform system (your appli
         --apply-immediately
     ```
 
-  To read more on how to modify the AUTH token see [AWS documentation][AWS Elasticache AUTH]
+  To read more on how to modify the AUTH token see [AWS documentation][AWS Elasticache AUTH]  
+### Upgrading Redis Engine
 
+  > **Important:**
+  > Upgrading your Redis engine version will incur a period of **downtime**, see the AWS documentation [here](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/VersionManagement.html) for details.
+  > Please take note that if you are upgrading from a version earlier than `5.0.5` then DNS propagation may increase downtime (AWS suggests this may be 30 seconds to 1 minute)
+
+  Update your `resources/elasticache.tf` resource to set your desired engine version and corresponding parameter group. If this is not set explicitly in your module call, the Elasticache module currently defaults to engine version [`5.0.6`](https://github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster/blob/main/variables.tf#L46).
+
+  ```
+  module "redis_cluster" {
+    ...
+    engine_version          = "6.x"
+    parameter_group_name    = "default.redis6.x"
+    ...
+  }  
+  ```  
+  If you planning on rotating your Elasticache AUTH token at the point of upgrade process, you can now continue with Step 1 above, since the upgrade and token rotation can be applied in a single PR.
```

Comment:
```suggestion
  If you are planning on rotating your Elasticache AUTH token at the point of the upgrade process, you can now continue with Step 1 above, since the upgrade and token rotation can be applied in a single PR.
```